reflex/app.py

```diff
@@ -102,60 +102,78 @@ class OverlayFragment(Fragment):
 
 
 class App(Base):
-    """A Reflex application."""
+    """The main Reflex app that encapsulates the backend and frontend.
 
-    # A map from a page route to the component to render.
-    pages: Dict[str, Component] = {}
+    Every Reflex app needs an app defined in its main module.
 
-    # A list of URLs to stylesheets to include in the app.
-    stylesheets: List[str] = []
+    ```python
+    # app.py
+    import reflex as rx
 
-    # The backend API object.
-    api: FastAPI = None  # type: ignore
+    # Define state and pages
+    ...
 
-    # The Socket.IO AsyncServer.
-    sio: Optional[AsyncServer] = None
-
-    # The state class to use for the app.
-    state: Optional[Type[BaseState]] = None
+    app = rx.App(
+        # Set global level style.
+        style={...},
+        # Set the top level theme.
+        theme=rx.themes.theme(accent_color="blue"),
+    )
+    ```
+    """
 
-    # Class to manage many client states.
-    _state_manager: Optional[StateManager] = None
+    # The global [theme](https://reflex.dev/docs/styling/theming/#theme) for the entire app.
+    theme: Optional[Component] = themes.theme(accent_color="blue")
 
-    # The styling to apply to each component.
+    # The [global style](https://reflex.dev/docs/styling/overview/#global-styles}) for the app.
     style: ComponentStyle = {}
 
-    # Middleware to add to the app.
-    middleware: List[Middleware] = []
-
-    # List of event handlers to trigger when a page loads.
-    load_events: Dict[str, List[Union[EventHandler, EventSpec]]] = {}
-
-    # Admin dashboard
-    admin_dash: Optional[AdminDash] = None
+    # A list of URLs to [stylesheets](https://reflex.dev/docs/styling/custom-stylesheets) to include in the app.
+    stylesheets: List[str] = []
 
-    # The async server name space
-    event_namespace: Optional[EventNamespace] = None
+    # A component that is present on every page.
```

Comment:
```suggestion
    # A component that is present on every page (defaults to the Connection Error banner).
```

And showing the default here in the API ref doesn't seem to be that helpful because it renders as like `<function ...>`. It would be awesome if we could get it to render like "rx.fragment(rx.connection_pulser(), rx.connection_modal())"; though not sure the best way at the moment

Reply:
Yeah I have a follow up ticket to improve the actual rendering of everything on the API reference page.